aws-lambda/dotnet/src/function.cs

```diff
@@ -0,0 +1,62 @@
+using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.Core;
+using Amazon.S3;
+using OpenTelemetry;
+using OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation;
+using OpenTelemetry.Trace;
+using System;
+using System.IO;
+using System.Text.Json;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+
+namespace Example
+{
+    public class Response
+    {
+        public string traceId { get; set; }
+    }
+
+    public class Function
+    {
+        public static TracerProvider tracerProvider;
+
+        static Function()
+        {
+            AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
+
+            tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
Lambda ignorance here again, is there a standard practice for managing the lifetime of static objects? In the case of a the tracer provider making sure it has an opportunity to get disposed would cause any telemetry that's been batched up to be sent.